lib/beaker/options/parser.rb

```diff
@@ -67,25 +60,25 @@ def split_arg arg
       #                         no .rb files are found overall
       def file_list(paths)
         files = []
-        if not paths.empty?
+        unless paths.empty?
           paths.each do |root|
             if File.file?(root)
               files << root
             elsif File.directory?(root) #expand and explore
               discover_files = Dir.glob(
-                File.join(root, "**/*.rb")
+                  File.join(root, "**/*.rb")
               ).select { |f| File.file?(f) }
               if discover_files.empty?
-                parser_error "empty directory used as an option (#{root})!"
+                @validator.parser_error "empty directory used as an option (#{root})!"
```

Comment:
These appear to be actual errors in the parsing.  If this is the case, why doesn't the parser just throw parser errors rather than validator's parser errors?